osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -195,15 +196,36 @@ def execute(extraction_order_id, execution_configuration):
     extraction_order.state = models.ExtractionOrderState.WAITING
     extraction_order.save()
 
+    result_directory_name = str(extraction_order.id) + '_' + NAME
+    result_directory_path = os.path.join(settings.RESULT_MEDIA_ROOT, result_directory_name)
+
     with tempfile.TemporaryDirectory() as tmp_dir:
         original_cwd = os.getcwd()
 
         try:
-            shutil.copyfile(
-                os.path.join(os.path.dirname(__file__), 'blackbox', 'docker-compose-conversion-blackbox.yml'),
-                os.path.join(tmp_dir, 'docker-compose.yml')
+            # Create a directory for result files of the current order inside the result media storage
+            #
+            # After finishing the conversion, create_output_file() will collect all files in this directory
+            # and attach them to the order.
+            # If the files were put in the root directory of the result media storage, they would be collected
```

Comment:
As it is not guaranteed that _all_ files would be collected by another task, should we replace 'they would be collected' with 'they might be collected', 'they would often be collected', 'they would probably be collected' or 'they can be collected'?